src/main/java/chronosacaria/mcda/registry/LootRegistry.java

```diff
@@ -0,0 +1,144 @@
+                    supplier.pool(poolBuilder);
+                    break;
+                case "entities/evoker":
+                    poolBuilder = FabricLootPoolBuilder.builder();
+                    addArmorSet(poolBuilder, ArmorSets.EVOCATION,0.10F);
+                    supplier.pool(poolBuilder);
+                    break;
+                case "blocks/blue_ice":
+                    poolBuilder = FabricLootPoolBuilder.builder()
+                            .rolls(BinomialLootTableRange.create(1, 0.05F))
+                            .with(ItemEntry.builder(ItemID.FROST_CRYSTAL));
+                    supplier.pool(poolBuilder);
+                    break;
+            }
+
+            if (BASTION_LOOT_TABLES.contains(id)) {
+                poolBuilder = FabricLootPoolBuilder.builder();
+                addArmorSet(poolBuilder, ArmorSets.GHOSTLY,       0.05F);
+                addArmorSet(poolBuilder, ArmorSets.GHOST_KINDLER, 0.01F);
+                supplier.pool(poolBuilder);
+            } else if (NETHER_FORTRESS_LOOT_TABLES.contains(id)) {
+                poolBuilder = FabricLootPoolBuilder.builder();
+                addArmorSet(poolBuilder, ArmorSets.GRIM, 0.05F);
+                supplier.pool(poolBuilder);
+            } else if (PILLAGER_TOWER_LOOT_TABLES.contains(id)) {
+                poolBuilder = FabricLootPoolBuilder.builder();
+                addArmorSet(poolBuilder, ArmorSets.DARK,  0.05F);
+                addArmorSet(poolBuilder, ArmorSets.THIEF, 0.05F);
+                supplier.pool(poolBuilder);
+            } else if (VILLAGE_SMITH_LOOT_TABLES.contains(id)) {
+                poolBuilder = FabricLootPoolBuilder.builder();
+                addArmorSet(poolBuilder, ArmorSets.PLATE,           0.05F);
+                addArmorSet(poolBuilder, ArmorSets.FULL_METAL,      0.01F);
+                addArmorSet(poolBuilder, ArmorSets.SNOW,            0.03F);
+                addArmorSet(poolBuilder, ArmorSets.WOLF,            0.05F);
+                addArmorSet(poolBuilder, ArmorSets.FOX,             0.01F);
+                addArmorSet(poolBuilder, ArmorSets.REINFORCED_MAIL, 0.05F);
+                addArmorSet(poolBuilder, ArmorSets.STALWART_MAIL,   0.01F);
+                supplier.pool(poolBuilder);
+            } else if (SUNKEN_SHIP_LOOT_TABLES.contains(id)) {
+                poolBuilder = FabricLootPoolBuilder.builder();
+                addArmorSet(poolBuilder, ArmorSets.SCALE_MAIL, 0.05F);
+                addArmorSet(poolBuilder, ArmorSets.MERCENARY,  0.05F);
+                supplier.pool(poolBuilder);
+            } else if (MINESHAFT_LOOT_TABLES.contains(id)) {
+                poolBuilder = FabricLootPoolBuilder.builder();
+                addArmorSet(poolBuilder, ArmorSets.SPELUNKER,    0.05F);
+                addArmorSet(poolBuilder, ArmorSets.CAVE_CRAWLER, 0.01F);
+                supplier.pool(poolBuilder);
+            }
```

Comment:
Both the `switch` and the chained `else if` are for performance - when it's one of them, it won't be another one, so there's no need to check the rest.
Previously it would check every single case separetely.